api/src/main/data/bt_language_names.xml

```diff
@@ -148,6 +159,14 @@ QUESTIONS?
 <aux_name0>Eastern Farsi</aux_name0>
 </language>
 <language>
+<id>94</id>
+<name>Dotyali</name>
+<doc>Dotyali</doc>
+<iso639-3>dty</iso639-3>
```

Comment:
Every language code needs an `iso639-1` element. Otherwise, the build fails. If a language doesn’t have an ISO 639-1 code, it gets the value `zz`.